<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>db_synchronizer_form</class>
 <widget class="QDialog" name="db_synchronizer_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>БД СИУ ---&gt; DynamicDocs Server (версия 0.1 beta)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Опрос каждые</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbTimer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbInterval">
       <item>
        <property name="text">
         <string>минут</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>часов</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>дней</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetTimer">
       <property name="text">
        <string>Задать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStopTimer">
       <property name="text">
        <string>Остановить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbQueryNow">
       <property name="text">
        <string>Опросить сейчас</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbMetadata">
     <property name="title">
      <string>Журнал обработки метаданных</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTextEdit" name="teLog"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="gbLogFile">
     <property name="title">
      <string>Журнал обработки файлов</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="teLogFile"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbSIUConnect">
       <property name="text">
        <string>Соединиться с БД СИУ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="gbFilter">
     <property name="title">
      <string>Фильтры</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="rbDate">
          <property name="text">
           <string>Начиная с указанной даты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="deDate"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSysDate">
        <property name="text">
         <string>Начиная с даты из системы (с даты последнего запуска конвертера)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbNoFilter">
        <property name="text">
         <string>Не фильтровать по дате</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
